src/icons/folderIcons.ts

```diff
@@ -604,6 +604,7 @@ export const folderIcons: FolderTheme[] = [
       { name: 'folder-quasar', folderNames: ['.quasar'] },
       { name: 'folder-cobol', folderNames: ['cobol'] },
       { name: 'folder-husky', folderNames: ['husky', '.husky'] },
+      { name: 'folder-storybook', folderNames: ['.storybook'] },
```

Comment:
Storybook setups now ship with a `stories` folder.  Maybe consider adding that too?

Reply:
Done 🙂